src/mongo/db/backup/backup_commands.cpp

```diff
@@ -0,0 +1,127 @@
+/*======
+This file is part of Percona Server for MongoDB.
+
+Copyright (c) 2006, 2016, Percona and/or its affiliates. All rights reserved.
+
+    Percona Server for MongoDB is free software: you can redistribute
+    it and/or modify it under the terms of the GNU Affero General
+    Public License, version 3, as published by the Free Software
+    Foundation.
+
+    Percona Server for MongoDB is distributed in the hope that it will
+    be useful, but WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public
+    License along with Percona Server for MongoDB.  If not, see
+    <http://www.gnu.org/licenses/>.
+======= */
+
+#include <boost/filesystem.hpp>
+
+#include "mongo/db/auth/action_type.h"
+#include "mongo/db/backup/backupable.h"
+#include "mongo/db/commands.h"
+#include "mongo/db/service_context.h"
+#include "mongo/db/storage/engine_extension.h"
+#include "mongo/db/storage/storage_options.h"
+
+namespace mongo {
+    extern StorageGlobalParams storageGlobalParams;
+}
+using namespace mongo;
+
+namespace percona {
+
+class CreateBackupCommand : public Command {
+public:
+    CreateBackupCommand() : Command("createBackup") {}
+    void help(std::stringstream& help) const override {
+        help << "Creates a hot backup, into the given directory, of the files currently in the "
+                "storage engine's data directory."
+             << std::endl
+             << "{ createBackup: <destination directory> }";
```

Comment:
Talking to @denis-protyvenskyi  this is documentation error, and we should be using  ```<< "{ createBackup: 1, "backupDir": <destionation directory> }";```